tap_woo/helpers/common_fields.py

```diff
@@ -0,0 +1,143 @@
+from singer_sdk import typing as th  # JSON Schema typing helpers
+
+METADATA_FIELD_SCHEMA = th.Property(
```

Comment:
Pulling these out into reusable blocks is a great idea, I'm a little annoyed I didnt think of it (and I fear I didnt think of it because I was relying on copilot to much to just write the schemas for me `:shame:`).  Nice work!